src/components/CouponDetails/CouponBulkActions.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Button, Form,
+} from '@edx/paragon';
+
+const CouponBulkActions = ({
+  value, options, disabled, onChange, handleBulkAction,
+}) => (
+  <div className="bulk-actions col-12 col-md-4 text-md-right mt-3 m-md-0 d-flex justify-content-end">
+    <Form.Group className="mt-1 flex-grow-1" controlId="bulkActions" name="bulk-actions">
+      <Form.Control
+        className="float-right w-50"
+        floatingLabel="Bulk action"
```

Comment:
Nit/wish: since this is brand new and a refactor and you already did it for ActionButton can we put all the actual text in a constants file? For my future translation wishes?